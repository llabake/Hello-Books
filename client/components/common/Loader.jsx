import React from 'react';

import ajaxLoader from '../../media/ajax-loader.gif';

const Loader = () => {
  return (
    <div
      className="center-align"
      style={{ marginTop: '1em', marginBottom: '1em' }}
    >
      <img src={ajaxLoader} alt="Loading..." />
    </div>
  )
}

export default Loader;