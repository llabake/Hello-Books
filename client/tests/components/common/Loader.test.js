import React from 'react';

import Loader from '../../../components/common/Loader';

const wrapper = shallow(<Loader />);

test('The Loader mounts correctly', () => {
  expect(wrapper).toMatchSnapshot();
})

