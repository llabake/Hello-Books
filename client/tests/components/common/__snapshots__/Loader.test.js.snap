// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Loader mounts correctly 1`] = `
<div
  className="center-align"
  style={
    Object {
      "marginBottom": "1em",
      "marginTop": "1em",
    }
  }
>
  <img
    alt="Loading..."
    src={null}
  />
</div>
`;
